dbt/models/shared_columns.md

```diff
@@ -1347,6 +1353,42 @@ See the [model-sales-val](https://github.com/ccao-data/model-sales-val)
 repository for a list of possible flags.
 {% enddocs %}
 
+## sv_outlier_reason1
+
+{% docs shared_column_sv_outlier_reason1 %}
+One of three possible reasons that a sale is
+flagged as on outlier. The priority for
+sv_outlier_reason$n column filling is
+ptax outlier > price outlier > characteric outlier.
+
+See the [model-sales-val](https://github.com/ccao-data/model-sales-val)
+repository for a list of possible flags.
+{% enddocs %}
+
+## sv_outlier_reason2
+
+{% docs shared_column_sv_outlier_reason2 %}
+One of three possible reasons that a sale is
+flagged as on outlier. The priority for
+sv_outlier_reason$n column filling is
+ptax outlier > price outlier > characteric outlier.
+
+See the [model-sales-val](https://github.com/ccao-data/model-sales-val)
+repository for a list of possible flags.
+{% enddocs %}
+
+## sv_outlier_reason3
+
+{% docs shared_column_sv_outlier_reason3 %}
+One of three possible reasons that a sale is
+flagged as on outlier. The priority for
+sv_outlier_reason$n column filling is
+ptax outlier > price outlier > characteric outlier.
+
+See the [model-sales-val](https://github.com/ccao-data/model-sales-val)
+repository for a list of possible flags.
+{% enddocs %}
```

Comment:
**[Nitpick, blocking]** Per my comment above, we can collapse these into one description given that they share the exact same text:

```suggestion
## sv_outlier_reason

{% docs shared_column_sv_outlier_reason %}
One of three possible reasons that a sale is
flagged as on outlier. The priority for
sv_outlier_reason$n column filling is
ptax outlier > price outlier > characteristic outlier.

See the [model-sales-val](https://github.com/ccao-data/model-sales-val)
repository for a list of possible flags.
{% enddocs %}
```